app/views/merchants/show.html.erb

```diff
@@ -0,0 +1,61 @@
+<div class="small-12 columns buttons_container row">
```

Comment:
Good work on responsiveness.  You could DRY up your views a bit by using partials, but well done!